spec/numberformat.html

```diff
@@ -724,7 +721,6 @@ <h1>SetNumberFormatUnitOptions ( _intlObj_, _options_ )</h1>
       </p>
       <emu-alg>
         1. Assert: Type(_intlObj_) is Object.
-        1. Assert: Type(_options_) is Object.
```

Comment:
Why is this assertion being removed?

Reply:
I can't find the comment now, but I believe @anba had objected to it after I added it in https://github.com/tc39/ecma402/pull/465, on the grounds of being unnecessary.

Reply:
Aren't assertions unnecessary by definition? Their purpose is to clarify invariants, and IMO having a type assertion for one input but not the other seems counterproductive to that end.